### DISCUSSION TITLE
Player Character not gaining Progress on Climax

### DISCUSSION THREAD

**Discussion by RevviBoi** [ErinaSugino/Starbound-SBR-Mod#62] [Trouble Shooting]
When activating a scene, everything plays normally, and the NPC actor can gain progress towards a climax. However, you cannot click the button to make them climax, and the player character does not gain any progress at all. This is the case for any race I attempt to do the scenes with, modded or vanilla.

Here's the log, thank you for any help you can provide!

[starbound.log](https://github.com/ErinaSugino/Starbound-SBR-Mod/files/13181982/starbound.log)

**Comment by ErinaSugino** [comment]
So, I'm not seeing any error related to climax - however, you did screw up your sextalk plugin config, resulting in the module erroring out constantly, spamming the log. This could be the reason.
